src/options.ts

```diff
@@ -33,6 +33,8 @@ export const getOptions = (
   // Only fallback to `jsconfig.json` when tsconfig can not be resolved AND custom tsconfig filename is not provided
   if (!result && !tsconfig) {
     result = getTsconfig(cwd, 'jsconfig.json');
+  } else {
+    console.warn('[rollup-plugin-swc] tsconfig does not exists, fallback to jsconfig.json');
   }
```

Comment:
```suggestion
  if (!result && !tsconfig) {
    console.warn('[rollup-plugin-swc] tsconfig does not exists, fallback to jsconfig.json');
    result = getTsconfig(cwd, 'jsconfig.json');
  }
```

Reply:
Also, IMHO we should use rollup built-in log functions here: https://rollupjs.org/plugin-development/#this-warn